src/client/Microsoft.Identity.Client/AuthenticationResultMetadata.cs

```diff
@@ -70,5 +70,13 @@ public AuthenticationResultMetadata(TokenSource tokenSource)
         /// Contains the Outcome of the region discovery if Region was used.
         /// </summary>
         public RegionDetails RegionDetails { get; set; }
+
+        /// <summary>
+        /// Contains telemetry data, formatted as JSON.
+        /// </summary>
+        /// <remarks>
+        /// Currently it is emitted only for desktop public client flows when the broker is used. 
+        /// </remarks>
+        public string TelemetryBlob { get; set; }
```

Comment:
nit: I still think BrokerTelemetry is better. It implies a certain schema and only one source. If this is too general and can have telemetry from multiple sources, what is the schema? How would the user parse it, look at it? Do we just concatenate a bunch of telemetry strings? And if we decide to expose telemetry from some other source, I don't think it'll be too bad to include a few more properties for each telemetry source.

Reply:
Yeah, I tend to agree. We already have a bunch of fields named "BrokerTag", "BrokerContext" etc. If we end up calling runtime even for non-broker flows, we can fix this (soft deprecate + hide and expose a new param)

Reply:
As long as this is a telemetry JSON blob schema can be figured out by the visualization and data processing pipelines. JSON as a format is not extremely difficult to consume and figure out the schema if humans read it. :)

I wouldn't expect apps to breakdown and process these blobs on client side rather they would send them to their telemetry pipelines and use the blob during deeper level analysis.

Reply:
This blob doesn't come from the broker directly and  depending on the platform the schema can substantially differ.